xarray/tests/test_computation.py

```diff
@@ -835,13 +836,24 @@ def test_dot(use_dask):
     assert actual.dims == ('b', )
     assert (actual.data == np.zeros(actual.shape)).all()
 
-    with pytest.raises(TypeError):
-        xr.dot(da_a, dims='a', invalid=None)
+    # Invalid cases
     with pytest.raises(TypeError):
         xr.dot(da_a.to_dataset(name='da'), dims='a')
     with pytest.raises(TypeError):
         xr.dot(dims='a')
 
+    # einsum parameters
+    xr.dot(da_a, da_b, dims=['b'], order='F')
+    actual = xr.dot(da_a, da_b, dims=['b'])
```

Comment:
I'm a little confused here -- the result of the first line is thrown away rather than checked, and the second line doesn't use parameters at all?

Reply:
@shoyer Sorry, you're right it did not make much sense. Fixed.